zebra-chain/src/parameters/network.rs

```diff
@@ -108,17 +170,62 @@ impl fmt::Display for Network {
 }
 
 impl Network {
+    /// Creates a new [`Network::Testnet`] with the default Testnet [`NetworkParameters`].
+    pub fn new_default_testnet() -> Self {
+        Self::Testnet(Arc::new(NetworkParameters::default()))
+    }
+
+    /// Creates a new configured [`Network::Testnet`] with the provided Testnet [`NetworkParameters`].
+    pub fn new_configured_testnet(params: NetworkParameters) -> Self {
+        Self::Testnet(Arc::new(params))
+    }
+
+    /// Returns true if the network is the default Testnet, or false otherwise.
+    pub fn is_default_testnet(&self) -> bool {
+        if let Self::Testnet(params) = self {
+            params.is_default_testnet()
+        } else {
+            false
+        }
+    }
+
+    /// Returns the [`NetworkKind`] for this network.
+    pub fn kind(&self) -> NetworkKind {
```

Comment:
I wanted to use the method so it's easier to find where the conversion is used. I was also just copying the pattern of [`io::Error.kind()`](https://doc.rust-lang.org/std/io/struct.Error.html#method.kind). There's also a `From` trait impl that's used for serde serialization of the owned `Network`.

I don't have a strong opinion if you'd prefer the `From` impl, but I have a weak preference for the `kind()` method otherwise.

Reply:
Sure, let's keep it as it is.